src/js/BookReader.js

```diff
@@ -2570,8 +2606,18 @@ exposeOverrideableMethod(BookModel, '_models.book', '_getDataProp');
  * @param {Object}
  */
 BookReader.prototype.updateFromParams = function(params) {
-  if ('undefined' != typeof(params.mode)) {
-    this.switchMode(params.mode);
+  // Set init, fragment change options for switchMode()
+  const {
+    mode = 0,
```

Comment:
is `0` a "natural" falsey considering mode is a number?

Reply:
Yes